pkgs/development/libraries/jemalloc/default.nix

```diff
@@ -47,10 +47,15 @@ stdenv.mkDerivation rec {
     ++ lib.optional (stdenv.isDarwin && stdenv.isx86_64) "--with-lg-vaddr=48"
   ;
 
-  env.NIX_CFLAGS_COMPILE = lib.optionalString stdenv.isDarwin "-Wno-error=array-bounds";
+  env = {
+    NIX_CFLAGS_COMPILE = lib.optionalString stdenv.isDarwin "-Wno-error=array-bounds";
+    # Tries to reference undefined shlibs
+    NIX_LDFLAGS = lib.optionalString (stdenv.cc.bintools.isLLVM && lib.versionAtLeast stdenv.cc.bintools.version "17") "--allow-shlib-undefined";
```

Comment:
is this necessary? `pkgsLLVM.jemalloc` builds fine for me without this change and the changes to checkFlags below.

Reply:
the errors in the bug report involve linking test which you don't do given `doCheck` is false, so still can't see how this flag is doing anything.

Reply:
Removed